scripts/ci-server.js

```
@@ -35,224 +183,1165 @@
+    case 'switch':
+      contextManager.switchContext(contextId);
+      res.json({ success: true, message: `Switched to context '${contextId}'` });
+      break;
+    
+    case 'reset':
+      contextManager.resetContext(contextId);
+      res.json({ success: true, message: `Context '${contextId}' reset` });
+      break;
+    
+    case 'list':
+      const contexts = Array.from(contextManager.contexts.keys());
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)